gutenberg/blocks/blocks.php

```diff
@@ -108,6 +129,9 @@ public function onInit() {
           "filter-language" => [
             'type' => 'string'
           ],
+          "field-config" => [
```

Comment:
field-config sounds a bit generic. What kind of data it is supposed to contain?

Reply:
It contains items for event list.  For example: Description, shortDescription, location, infoUrl. 
So WordPress admin user can choose what items are sent to event.php